sdk/react/upNext.js

```diff
@@ -130,8 +130,13 @@ var UpNext = React.createClass({
           source={{uri: this.props.nextVideo.imageUrl}}
           style={styles.thumbnail} >
           <TouchableHighlight style={styles.thumbnail}
-            onPress={this.clickUpNext}>
-            <Text style={styles.countdownText}>{this.props.config.icons.play.fontString}</Text>
+            onPress={this.clickUpNext}
+            accesible={true}
+            accessibilityLabel={"Play up next video"}>
```

Comment:
Can we add it as constant variable ? Something like "BUTTON_NAMES.DISMISS"

Reply:
Done